packages/edit-navigation/src/store/utils.js

```diff
@@ -7,6 +7,7 @@ import { keyBy, omit } from 'lodash';
  * WordPress dependencies
  */
 import { serialize } from '@wordpress/blocks';
+import { NEW_TAB_TARGET_ATTRIBUTE } from '@wordpress/menus';
```

Comment:
This constant should probably belong in the DOM package, but It might be better for it not to be a package export.

Either duplicating it or moving `blockAttributesToMenuItem` to the menus package would work.

Reply:
Removed